### DISCUSSION TITLE
Fail2ban not working

### DISCUSSION THREAD

**Discussion by fxp1303** [modoboa/modoboa#2163] [Q&A]
The fail2ban log is empty and doesn't seem to be working.

Maybe it didn't install and configure during installation, how do I retrofit fail2ban on a working Modoboa implementation?

Any assistance would be very much appreciated.

**Comment by tonioo** [comment]
fail2ban is not installed by default but you can add it after. You can also log failed connection attempts if you want to lock the access to the web interface (see https://github.com/modoboa/modoboa/issues/1359).

**Reply by fxp1303** [reply]
Many thanks, have installed and implemented suggestion from another post related to similar subject.

**Reply by fxp1303** [reply]
Many thanks for your kind assistance, please kindly mark discussion as resolved.
